test/fixtures/db_connections.yml

```diff
@@ -0,0 +1,23 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
```

Comment:
I guess, we are not going to use fixtures and is nowhere in use. So, we can delete this fixture file. :wink: :+1:

Reply:
🤣 yepa